README.md

```diff
@@ -107,6 +107,8 @@ Edit it by adding the following lines:
     Default value: AUTOGENERATED<br/>
     Example value: '{"Netflix": "11101200001", "YouTube": "111299001912", "Spotify": "3201606009684"}'<br/>
     Known lists of App IDs: [List 1](https://github.com/tavicu/homebridge-samsung-tizen/issues/26#issuecomment-447424879), [List 2](https://github.com/Ape/samsungctl/issues/75#issuecomment-404941201)<br/>
+    You can also add at the end of every app ID an additional value (separated by "/") that indicate the channel name exposed by SmartThings. This will help identifing the running app if SmartThings API are enabled. This additional value is optional and if not present will be used the app ID to identify the running app.<br/>
+    Example value: '{"Netflix": "11101200001/org.tizen.netflix-app", "Prime Video": "3201512006785/org.tizen.ignition", "Spotify": "3201606009684"}'<br/>
```

Comment:
why do we need this? with the current logic, if you set `"Netflix": "11101200001"` then that will be handled by the old http poll logic, if you set `"Netflix": "org.tizen.netflix-app"` then it will be handled by the SmartThings logic, and if you set all the IDs with `org.`, then the http polling logic simply will not be used at all. So where is the benefit of setting IDs like this `11101200001/org.tizen.netflix-app`?

Reply:
You are right at 99%.
But there are some rare case (as Prime Video) in which the ID of the app that is shown as channel is different from the ID used to call it. And also using the old http logic the attribute "visible" is always False. 
So with this option (that is optional, you can omitt the second part) we can manage this situation.

Reply:
So for Prime Video and a few other apps, the SmartThings ID won't work to start the app, while the old ID (with the numbers) works to start the app?

Reply:
Yes, that's it. Or in same case the SmartThings ID used to launch the app is different from the one shown as Channel.

Reply:
This is the only thing I can't justify, the level of added complexity, both to the code and to documentation comprehension is simply too high. I'm not saying this issue should not be solved, just that there must be _some_ better way of doing it.

Reply:
Maybe there is some way to start an app through the SmartThings API, which would work with the SmartThings ID.

Reply:
Is the only solution that I found, due to the fact that I use Prime and I don't like to have input always come back to generic "HDMI". I can agree with you on the complexity, but consider that is not requested to add the second part and who will not need will not use. Obviosly is ok is you have any other idea.

Reply:
Are you able to reproduce my issue on your TV? What do you see on SmartThings cloud when you launch Prime?

Reply:
I see `tvChannelName: "org.tizen.ignition"` (this is not out of the ordinary, all apps show as `tvChannelName` in the SmartThings API), but it does seem that if I try to run the app with `org.tizen.ignition` I get an error on my TV saying:
> Unable to connect to Samsung Server. Please try later. (AS400)

Reply:
The old ID does seem to work to start the app..

Reply:
I understand the issue now, but I still don't particularly like the way this was implemented.. I've been trying to run apps with the SmartThings API, but it doesn't seem to work (yet, at least), I tried with both the input source method and the channel method.

Reply:
I keep thinking of this issue and possible solutions for it, but I haven't been able to find any good solution yet. If I still fail to find some better way of handling it, I might accept it as it is.